exchangelib/autodiscover.py

```diff
@@ -429,7 +429,7 @@ def _parse_response(response, encoding='utf-8'):
     # There are three possible protocol types: EXCH, EXPR and WEB. EXPR is for EWS. See
     # http://blogs.technet.com/b/exchange/archive/2008/09/26/3406344.aspx
     for protocol in protocols:
-        if get_xml_attr(protocol, '{%s}Type' % RESPONSE_NS) != 'EXPR':
+        if get_xml_attr(protocol, '{%s}Type' % RESPONSE_NS) not in ('EXCH', 'EXPR'):
```

Comment:
Can you please explain why this was necessary?

I've had autodiscover running against Exchange 2010 servers for years, and according to my reading, `EXCH` contains data about the RPC protocol that we don't use. See https://blogs.technet.microsoft.com/exchange/2008/09/26/the-autodiscover-service-and-outlook-providers-how-does-this-stuff-work/

I suspect that you're dealing with an Exchange server with a broken configuration (since it apparently either doesn't contain an `EXPR` config, or `EXPR` contains wrong data), and that I fear that this change will break autodiscover subtly for other users since the code now arbitrarily selects either `EXPR` or `EXCH`.